XRTK-Core/Packages/com.xrtk.core/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -1007,25 +1006,49 @@ private async void PairDevice()
             }
         }
 
+        /// <summary>
+        /// Builds the Unity Project for the <see cref="EditorUserBuildSettings.activeBuildTarget"/>
+        /// </summary>
         public static async void BuildUnityProject()
         {
             Debug.Assert(!isBuilding);
             isBuilding = true;
 
-            appxCancellationTokenSource = new CancellationTokenSource();
-            await UwpPlayerBuildTools.BuildPlayer(BuildDeployPreferences.BuildDirectory, cancellationToken: appxCancellationTokenSource.Token);
-            appxCancellationTokenSource.Dispose();
-            appxCancellationTokenSource = null;
+            buildCancellationTokenSource = new CancellationTokenSource();
+
+            switch (EditorUserBuildSettings.activeBuildTarget)
+            {
+                case BuildTarget.WSAPlayer:
+                    await UwpPlayerBuildTools.BuildPlayer(BuildDeployPreferences.BuildDirectory, cancellationToken: buildCancellationTokenSource.Token);
+                    break;
+                case BuildTarget.Lumin:
+                    LuminPlayerBuildTools.BuildPlayer(new BuildInfo());
```

Comment:
I would have expected this call to be an override of the base BuildUnityPlayer call instead of hoping out and back again.